### PR TITLE
[NetManager] Site activities visualization

### PR DESCRIPTION
**Pull request by danielmarv**
#### Summary of Changes (What does this PR do?)

- I have created ActivitiesLogs folder in the redux to handle the call for the site activities.
- i have created the Activities folder in the component to handle the site activities page functionality
- i have also added a link on the network monitoring Menu  

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state

#### What are the relevant tickets?

- [AN-492)](https://airqoteam.atlassian.net/browse/AN-492)

#### Screenshots (optional)
siteActivities1
siteActivities2

### CODE REVIEW

File: netmanager/src/views/components/Activities/ActivitiesTable.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    if (isEmpty(site_activities)) {
+      setIsLoading(true);
+      if (!isEmpty(activeNetwork)) {
+        dispatch(loadActivitiesData(activeNetwork.net_name));
+      }
+      setIsLoading(false);
+    }
+  }, []);
+
+  // const handleDeleteSite = (e) => {
```
Comment (Baalmart) [inline netmanager/src/views/components/Activities/ActivitiesTable.js:50]: try to minimise commented out code. @danielmarv

File: netmanager/src/views/layouts/common/Sidebar/Sidebar.js [hunk 2]
```diff
@@ -117,7 +117,8 @@ const allMainPages = [
     nested: true,
     nestItems: [
       { title: 'Network Map', href: '/manager/map' },
-      { title: 'Network Statistics', href: '/manager/stats' }
+      { title: 'Network Statistics', href: '/manager/stats' },
+      { title: 'Site Activities', href: '/manager/activities' }
```
Comment (Baalmart) [inline netmanager/src/views/layouts/common/Sidebar/Sidebar.js:121]: We could rename this from Site Activities to **Site Activity Logs** @danielmarv

**Review by Baalmart — COMMENTED** [review]
Thanks @danielmarv for this.
On a side note, just been seeing many CORs related errors recently. We shall investigate separately.

I have also left a few comments/suggestions in the PR, please locate them and respond accordingly.

**Comment by Baalmart** [comment]
Screenshot 2023-07-13 at 23 18 57
The CORS errors I have been referring to. @danielmarv

**Comment by danielmarv** [comment]
am waiting for the deploy preview

File: netmanager/src/views/components/Activities/ActivitiesToolbar.js [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+      </Dialog>
+    </>
+  );
+};
+
+ActivitiesToolbar.propTypes = {
+  className: PropTypes.string
+};
+
+export default ActivitiesToolbar;
```
Comment (Codebmk) [inline netmanager/src/views/components/Activities/ActivitiesToolbar.js:273]: @danielmarv what is this code for? I dont see any form in the UI

Comment (danielmarv) [reply netmanager/src/views/components/Activities/ActivitiesToolbar.js:273]: ok @Codebmk

Comment (danielmarv) [reply netmanager/src/views/components/Activities/ActivitiesToolbar.js]: This is resolved @Codebmk

File: netmanager/src/views/layouts/common/Sidebar/Sidebar.js [hunk 4]
```diff
@@ -117,7 +117,8 @@ const allMainPages = [
     nested: true,
     nestItems: [
       { title: 'Network Map', href: '/manager/map' },
-      { title: 'Network Statistics', href: '/manager/stats' }
+      { title: 'Network Statistics', href: '/manager/stats' },
+      { title: 'Site Activity Logs', href: '/manager/activities' }
```
Comment (Baalmart) [inline netmanager/src/views/layouts/common/Sidebar/Sidebar.js:121]: @Codebmk , @danielmarv , perhaps we could call it `Network Activity Logs`?
Just thought about it this morning. :)
Just to help maintain the flow of things from the rest of the item in sidebar popup.

Thanks!

Comment (danielmarv) [reply netmanager/src/views/layouts/common/Sidebar/Sidebar.js]: @Baalmart it is resolved

File: netmanager/src/redux/ActivityLogs/operations.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        });
+      })
+      .catch((err) => {
+        dispatch({
+          type: LOAD_ACTIVITIES_SUMMARY_FAILURE,
+          payload: err
+        });
+      });
+  };
+};
```
Comment (Codebmk) [inline netmanager/src/redux/ActivityLogs/operations.js:57]: @danielmarv where is the activity_details redux action used? Seems like there's a lot of new unused code.

Comment (danielmarv) [reply netmanager/src/redux/ActivityLogs/operations.js:57]: i had just left it for future use, but ok let me remove them

Comment (Codebmk) [reply netmanager/src/redux/ActivityLogs/operations.js:57]: Just remove all the unused code @danielmarv

Comment (Codebmk) [reply netmanager/src/redux/ActivityLogs/operations.js:57]: Are you using the loadActivitiesSummary operation?

**Review by Codebmk — APPROVED** [review]
LGTM thanks @danielmarv

**Pull request merged**